src/adapter/src/catalog/builtin_table_updates/notice.rs

```diff
@@ -68,31 +40,39 @@ impl CatalogState {
             }
         }
 
+        // These notices will be persisted in a system table, so should not be
+        // relative to any user's session.
+        let conn_catalog = self.for_system_session();
```

Comment:
I didn't know about `for_system_session()` which is why I think I implemented `ExprHumanizer` for `CatalogState`.